ors-engine/src/main/java/org/heigit/ors/exceptions/ParameterOutOfRangeException.java

```diff
@@ -18,16 +18,18 @@
 public class ParameterOutOfRangeException extends StatusCodeException {
     private static final long serialVersionUID = 7728944138955234463L;
 
+    private static final String PARAMETER_KEY = "Parameter '";
+
     public ParameterOutOfRangeException(int errorCode, String paramName) {
-        super(StatusCode.BAD_REQUEST, errorCode, "Parameter '" + paramName + "' is out of range.");
+        super(StatusCode.BAD_REQUEST, errorCode, PARAMETER_KEY + paramName + "' is out of range.");
     }
 
     public ParameterOutOfRangeException(int errorCode, String paramName, String customMessage) {
-        super(StatusCode.BAD_REQUEST, errorCode, "Parameter '" + paramName + "' is out of range: " + customMessage);
+        super(StatusCode.BAD_REQUEST, errorCode, PARAMETER_KEY + paramName + "' is out of range: " + customMessage);
```

Comment:
i would not put the space into the PARAMETER_KEY but either do 
`PARAMETER_KEY + " " +...`
or use string formatting in the first place